File: Media10n.py
#Declara variável
s = 0

#Cria um loop de 10 interações, pede 10 números e soma eles
for i in range(10):
    n1 = int(input("Digite um número: "))
    s += n1

#Divide o valor por 10 e mostra o resultado
s /= 10
print(f"A média é {s}")

File: Media10n.c
#include <stdio.h>

int main(){

    //Declara as variáveis
    int i;
    int s = 0;

    //Cria um loop de 10 interações, que pede um número e soma eles
    for (i = 1; i < 11; i++) {
        printf("Digite um número nº%d\n", i);
        int n;
        scanf("%d", &n);
        s += n;
        printf("A soma é igual a: %d\n", s);
    }

    //Cálcula a média e mostra o resultado
    int m = s /10;
    printf("A média é: %d\n", m);
    return 0;
}